site/src/components/card/Card.module.css

```diff
@@ -1,5 +1,4 @@
 .card {
```

Comment:
Can we combine this `.card` block with the one immediately below?

Reply:
We definitely can but I believe the general css variables convention is to declare them in a separate block

Reply:
Ah Ok. I wasn't aware of that convention. In that case leave it as is.